deployments/examples/ocis_keycloak/config/keycloak/clients/cyberduck.json

```diff
@@ -0,0 +1,63 @@
+{
+    "clientId": "3keLfua0olYvW1zKXTDB3OjAMPEYWEQNuiscli395GKJOiPnPURNQWGvGCJZf4Hw",
+    "name": "Cyberduck",
+    "surrogateAuthRequired": false,
+    "enabled": true,
+    "alwaysDisplayInConsole": false,
+    "clientAuthenticatorType": "client-secret",
+    "secret" : "yoqICbLIeYbpZPqDH4D8k4NKb04HqnrWBntEeVZEQ5gO1RmaUlln0Aqu1dj2UoF4",
+    "redirectUris": [
+        "x-cyberduck-action:oauth",
+        "x-mountainduck-action:oauth"
```

Comment:
```suggestion
        "x-cyberduck-action://oauth",
        "x-mountainduck-action://oauth"
```

Reply:
Or alternatively reopen #364.

Reply:
`x-cyberduck-action:oauth` works fine in Keycloak with Infinite Scale. Only ownCloud oauth2 app needs `x-cyberduck-action://oauth`. What’s the preferred way to open Cyberduck for this?

Reply:
The preferred is `x-cyberduck-action:oauth`. We only introduced [^1] support to have redirect URIs with an (empty) authority component preceded by two slashes (`//`) because of the same issue with Microsoft Azure. 

[^1]: https://github.com/iterate-ch/cyberduck/pull/14880/commits/0b2e8c50754b6338c16285e7b93cac3088065821